workspaces/liferay-customer-workspace/client-extensions/liferay-customer-etc-spring-boot/src/main/java/com/liferay/customer/JiraRestController.java

```diff
@@ -37,23 +38,59 @@
 @RestController
 public class JiraRestController extends BaseRestController {
 
-	public JiraRestController() {
+	public JiraRestController(
+		@Value(
+			"${liferay.customer.jira.security.vulnerability.field.affected.versions.details}"
+		)
+		String affectedVersionsDetails,
+		@Value("${liferay.customer.jira.security.vulnerability.field.category}")
+			String category,
```

Comment:
should `String category` and some of the other params have a different indentation than the rest? it looks kind of odd.

Reply:
source formatter did it, I can't indent it in any other way

Reply:
Are the private fields not populated yet so we have to do this? if so, can you remove the values on the private fields and set it in the constructor?

Reply:
It's because the String[] _securityVulnerabilitiesIssueFields is a final or static variable. (Source formatter will add "final".) It will always be empty. Solution is either in the constructor or adding setter methods.

Re-pushed so we set the value in the constructor only.

Reply:
_securityVulnerabilitiesIssueFields  is still a final variable and only set once in the constructor. What does moving the @value to constructor params from the private fields do?